<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save_mission_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Mission</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_task_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_task_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_task_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Task</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="param_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="param_val_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_param_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="param_type">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="param_val">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_param_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Parameter</string>
    </property>
   </widget>
   <widget class="QListView" name="param_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="task_list">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="param_list_view">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter List</string>
    </property>
   </widget>
   <widget class="QLabel" name="task_list_view">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Task List</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>21</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>21</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMission_Planner">
    <property name="title">
     <string>Mission Planner</string>
    </property>
   </widget>
   <addaction name="menuMission_Planner"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
